src/components/PetForm.js

```diff
@@ -19,7 +19,7 @@ export default class PetForm extends Component {
       breeds: [],
       name: '',
       typeOfAnimal: 'Cat',
-      breed: '',
+      breed: 'Domestic ShortHair',
```

Comment:
I also fixed this in PR #15 and it'll be dynamic by the breeds loaded. You can also remove this change to prevent merge conflicts.